ios/MobiusMobileWallet/Info.plist

```diff
@@ -95,6 +95,17 @@
 		<string>Roboto Medium.ttf</string>
 		<string>Roboto.ttf</string>
 		<string>Helvetica Neue Italic.ttf</string>
+		<string>Entypo.ttf</string>
```

Comment:
Minor thing: I think all of this fonts are not using in the app and must remove from the bundle.